todo.py

```diff
@@ -5,9 +5,24 @@ def __init__(self):
         self.tasks = []
 
     def add_task(self, task):
-        """Adds a new task to the list."""
-        self.tasks.append(task)
-        print(f'Task added: {task}')
+        """Adds a new task to the list if it doesn't already exist."""
+        if self.task_exists(task):
+            print(f"Task '{task}' already exists in the list! Cannot add duplicate tasks.")
```

Comment:
It would be useful to include the task number of the duplicate in this message

Reply:
Task number is added!